Status.sh

```diff
@@ -47,7 +47,7 @@ swap="$(free -h | sed -n 3p | awk '{print $2}')"
 
 #Ubuntu version
 ubuntu="$(lsb_release -a | sed -n 2p | awk '{print $3, $4}')"
-if [ ! "$ubuntu" = "20.04.5 LTS" && ! "$ubuntu" = "20.04.6 LTS" ]
+if [ ! "$ubuntu" = "20.04.6 LTS" ]
```

Comment:
wouldn't that make 20.04.5 systems as deprecated?
Should they always be updated once we reach here?

Reply:
The only time someone gets to this line is after they update their platform.
This line updates the platform:
https://github.com/Navid200/cgm-remote-monitor/blob/9a473c83e65d2a37d45840a97bb8f0ca9e64f055/menu_GC_Setup.sh#L14

That line is a pointer to this section:
https://github.com/Navid200/cgm-remote-monitor/blob/9a473c83e65d2a37d45840a97bb8f0ca9e64f055/menu_GC_Setup.sh#L31-L45

This line from that section is the line that updates all the scripts including the Status page utility.
https://github.com/Navid200/cgm-remote-monitor/blob/9a473c83e65d2a37d45840a97bb8f0ca9e64f055/menu_GC_Setup.sh#L39

And the next line of the same section updates packages, including Ubuntu.
https://github.com/Navid200/cgm-remote-monitor/blob/9a473c83e65d2a37d45840a97bb8f0ca9e64f055/menu_GC_Setup.sh#L40


If they update their platform, they will upgrade their Ubuntu to 20.04.6.  In that case, they will never have 20.04.5.

Those who are running 20.04.5 can continue to do that.  They will never see this marker on their status page because their status page has not been updated yet.